gluten-core/src/main/scala/io/glutenproject/expression/UnaryExpressionTransformer.scala

```diff
@@ -60,6 +58,38 @@ class NormalizeNaNAndZeroTransformer(
   }
 }
 
+class PromotePrecisionTransformer(child: ExpressionTransformer, original: PromotePrecision)
```

Comment:
seems `PromotePrecisionTransformer` is a normal transformer, maybe not to add a new one ?

Reply:
it‘s a custom transformer, it direct use clild transform instead of itsself

Reply:
Got it.